pyproject.toml

```diff
@@ -60,6 +60,9 @@ minio = [
   "django-minio-storage>=0.5",
   "minio>=7",
 ]
+pydantic = [
+  "pydantic>=2",
```

Comment:
Should we depend on Django-Ninja instead, given that's how we practically anticipate this will be used?

The counterargument is that this unnecessarily locks us into Django-Ninja and makes it harder to use otherwise-compatible replacements like Django Shinobi.